Formula/b/build2.rb

```diff
@@ -0,0 +1,149 @@
+class Build2 < Formula
+  desc "C/C++ Build Toolchain"
+  homepage "https://build2.org"
+  url "https://download.build2.org/0.16.0/build2-toolchain-0.16.0.tar.xz"
+  sha256 "23793f682a17b1d95c80bbd849244735ed59a3e27361529aa4865d2776ff8adc"
+  license "MIT"
+
+  # TODO: add livecheck based on https://download.build2.org/toolchain.sha256
+  #       for background, see https://download.build2.org/README
+
+  # Note that for now we use SQLite from Mac OS instead of Homebrew at the
+  # insistence of Homebrew maintainers but may switch in the future if have a
+  # good enough reason (for example, higher SQLITE_MAX_ATTACHED limit). See PR
+  # #141550 for details. Note also that on Linux it comes from Homebrew so
+  # we have to arrange for library search paths, etc (see below).
+  #
```

Comment:
This is all in the PR, no need to have it in the file as well

Reply:
Dropped.